addon/components/o-s-s/chip.ts

```diff
@@ -0,0 +1,66 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+
+export type SkinType = 'default' | 'primary' | 'success' | 'danger';
+
+type SkinDefType = {
+  [key in SkinType]: string;
+};
+
+export const SkinDefinition: SkinDefType = {
+  default: 'default',
+  primary: 'primary',
+  success: 'success',
+  danger: 'danger'
+};
+
+const BASE_CLASS_CHIP: string = 'upf-chip';
+interface OSSChipArgs {
+  skin?: string;
+  label?: string;
+  onClick(crossClick: boolean): void;
```

Comment:
I think `onRemove` would be better/more precise given that it's the click on the "cross" only and not on the whole chip